fuzzers/aflchurn/fuzzer.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for AFLChurn fuzzer."""
+
+import shutil
+import subprocess
+import os
+
+from fuzzers import utils
+
+
+# '-fsanitize-coverage=trace-pc-guard'
+def prepare_build_environment():
+    """Set environment variables used to build targets for AFL-based
+    fuzzers."""
+    cflags = ['-fsanitize=address', '-fsanitize-address-use-after-scope']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cflags)
+    utils.append_flags('ASAN_OPTIONS', ['abort_on_error=1', 'symbolize=0'])
+
+    os.environ['CC'] = '/afl/afl-clang-fast'
+    os.environ['CXX'] = '/afl/afl-clang-fast++'
+    os.environ['FUZZER_LIB'] = '/libAFL.a'
+
+    os.system(
+        'apt-get install -y python-software-properties software-properties-common && \
+        add-apt-repository ppa:git-core/ppa -y && \
+        apt-get update && \
+        apt-get install git -y')
```

Comment:
Since AFLChurn uses `git` to get the commit information, is there anywhere else that updates the git?
In our experiments, we notice that old `git` may fail (our git version is 2.17.1).

Reply:
git should be pretty recent, if not, just add this in Dockerfile, not fuzzer.py